exec/java-exec/src/main/java/org/apache/drill/exec/ops/FragmentContext.java

```diff
@@ -245,6 +245,16 @@ public SchemaPlus getRootSchema() {
   }
 
   /**
+   * Returns the statement type (e.g. SELECT, CTAS, ANALYZE) from the query context.
+   * @return query statement type {@link QueryContext.StatementType}, if known.
+   */
+  public QueryContext.StatementType getStatementType() {
+    if (queryContext != null) {
```

Comment:
Please explain under what conditions the query context will be null. Is this just being paranoid? Or, are there statement types where there is no context? And, if there are such statements, aren't those statements known, not "UNKNOWN"?